files/default/supervisor-ckan-harvest-fetch.conf

```diff
@@ -0,0 +1,47 @@
+[unix_http_server]
+file=/var/tmp/supervisor.sock
+
+[rpcinterface:supervisor]
+supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
+
+; ===================================================================
+; Supervisor configuration for CKAN Harvest background fetch consumer
+; ===================================================================
+
+; 1. Copy this file to /etc/supervisor/conf.d
+; 2. Make sure the paths below match your setup
+
+
+[program:ckan-harvest-fetch]
+
+; Use the full paths to the virtualenv and your configuration file here.
+command=/usr/lib/ckan/default/bin/paster --plugin=ckan-harvest harvester fetch_consumer --config=/etc/ckan/default/production.ini
+
+
+; User the fetch consumer runs as.
+user=ckan
+
+
+; Start just a single fetch consumer.
+numprocs=1
+process_name=%(program_name)s-%(process_num)02d
+
+
+; Log files.
+stdout_logfile=/var/log/ckan/ckan-harvest-fetch.log
```

Comment:
add note that these needs to by copied to s3 and setup to cloudwatch logs inside opsworks